services/reconciler/createChangeSetFromCompareStructures.server.ts

```
@@ -36,7 +36,7 @@ export const createRemovedChangeSetFromCompareStructures = ({
         Action: 'DELETE',
         ResourceRecordSet: {
           Name: fqdn,
-          Type: type,
+          Type: type as RRType,
```

Instead of using `as`, which disables the compiler, can we add a function to map from type to RRType?

---

@humphd Sure! Do we want to throw an error in case of conversion failure?

---

Absolutely, since that would mean there's a bug.